src/clang-cpp-frontend/clang_cpp_convert_bind.cpp

```diff
@@ -28,31 +28,47 @@ CC_DIAGNOSTIC_POP()
 #include <clang-cpp-frontend/clang_cpp_convert.h>
 #include <util/expr_util.h>
 
-bool clang_cpp_convertert::perform_virtual_dispatch(const clang::Decl &decl)
+bool clang_cpp_convertert::perform_virtual_dispatch(
+  const clang::MemberExpr &member)
 {
+  // x.F() can't be a virtual dispatch
+  if(!member.isArrow())
+    return false;
+
+  const clang::Decl &decl = *member.getMemberDecl();
   switch(decl.getKind())
   {
   // TODO: dtor might be virtual too
   //case clang::Decl::CXXDestructor:
   case clang::Decl::CXXMethod:
   {
+    const clang::CXXMethodDecl &cxxmd =
+      static_cast<const clang::CXXMethodDecl &>(decl);
+
+    clang::LangOptions langOpts;
+    langOpts.CPlusPlus = 1;
+    langOpts.RTTI = 1;
```

Comment:
Why do you assign `1` here?

Reply:
To set LangOptions in C++ mode with RTTI enabled.

Added this explanation in the code as a comment.